### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by LucaNicosia**
This PR adds a template for new PRs.
The first line indicates to set the PR target to `develop` branch

### CODE REVIEW

**Review by claudious96 — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### Set `develop` as target branch of this PR (delete this line before issuing the PR)
+
+## Description
```
Comment (ddutt) [inline .github/pull_request_template.md:3]: Please state that the test inclusion requirements as follows:

* any fix or enhancement SHOULD include relevant new tests or test updates, if any tests need updating. 
* a new platform support MUST include the relevant input files similar to what we have in tests/integration/sqcmds/*-input directories, along with the relevant tests in the tests/integration/sqcmds/*-samples dir. That list MUST include the all.yml file fully filled out.
* any new service (or table) addition MUST include comments about what network OS are supported (along with version) with this command along with test samples for those platforms and input files in the *-input dir

Comment (ddutt) [reply .github/pull_request_template.md:3]: Please provide similar comments about what's necessary in documentation.

**Review by ddutt — CHANGES_REQUESTED** [review]

File: docs/developer/contribute.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+To add your functionality to SuzieQ, you need to follow some steps:
+
+1. First of all, fork the SuzieQ repository and clone the forked project on your computer
```
Comment (claudious96) [inline docs/developer/contribute.md:3]:
```suggestion
1. Fork the SuzieQ repository and clone the forked project on your local machine
```

File: docs/developer/contribute.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        All the commits must be signed-off using the `-s` parameter of the git command.
+
+    !!!Warning
+        In case the newly developed code contains an enhancement or a new platform/service support, some tests have to be added:
+
+        - any fix or enhancement SHOULD include relevant new tests or test updates, if any tests need updating.
+        - a new platform support MUST include the relevant input files similar to what we have in tests/integration/sqcmds/-input directories, along with the relevant tests in the tests/integration/sqcmds/-samples dir. That list MUST include the all.yml file fully filled out.
+        - any new service (or table) addition MUST include comments about what network OS are supported (along with version) with this command along with test samples for those platforms and input files in the *-input dir
+
+3. Once you have done with coding (and testing), open a pull request from the forked github page.
```
Comment (claudious96) [inline docs/developer/contribute.md:18]:
```suggestion
3. Once you're done coding the feature and the tests, open a pull request from the forked github page.
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (LucaNicosia) [reply .github/pull_request_template.md:10]: Done

**Pull request merged**